src/components/elements/DidsByDomain.tsx

```diff
@@ -0,0 +1,47 @@
+import { Box, Text } from "rimble-ui";
+import { DidView, DomainImage } from "./";
+import { H5 } from "../layouts";
+import { colors } from "../../themes";
+
+export interface DidsByDomainProps {
+  didDocs: any[];
+  didCopy?: boolean;
+  domain: string;
+  linkDomain?: string;
+  name?: string;
+}
+
+export const DidsByDomain: React.FunctionComponent<DidsByDomainProps> = (props) => {
```

Comment:
this is almost identical to DidByDomain, with a few modifications, but I created a new component since I'm not sure if these updates would cause issues with the Agent UI

Reply:
You can merge these two I'll add the didDocs on Agent. They're using the same endpoint so it shouldn't be an issue.